### PR TITLE
PXB-2127 xbcloud fails to upload backups with empty database to min.io storage

### PR DESCRIPTION
**Pull request by rahulmalik87**
Problem:
Xbcloud fails to upload data in minio with empty database.

Analysis:
xbcloud is placing dummy file /db.opt in an empty directory.
Path becomes /path/to//db.opt instead of /path/to/db.opt
minio assue path of the file as "/db.opt" and throws error
unspported file name.

Fix:
intead of /db.opt use db.opt as output file

### CODE REVIEW

File: storage/innobase/xtrabackup/src/backup_copy.cc [hunk 1]
```diff
@@ -1397,7 +1397,7 @@ backup_files(const char *from, bool prep_mode)
 			/* backup fake file into empty directory */
 			char path[FN_REFLEN];
 			ut_snprintf(path, sizeof(path),
-					"%s/db.opt", node.filepath);
+					"%sdb.opt", node.filepath);
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/src/backup_copy.cc]: this is a bit risky. What if the path doesn't end with trailing slash? My suggestion is to keep this %s/db.opt thing.  Add code to remove trailing slash in path.

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
Regarding testcase, any existing test to cover?

**Comment by satya-bodapati** [comment]
Also please mention that "//" causes problems with minio S3 blah blah.. in the commit message.

**Comment by satya-bodapati** [comment]
And also add the reason for this fake file in empty dir in commit msg

**Comment by satya-bodapati** [comment]
Jenkins test link plz

**Comment by rahulmalik87** [comment]
> Regarding testcase, any existing test to cover?

added the testcase

**Pull request closed without merging**